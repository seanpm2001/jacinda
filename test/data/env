TERM_SESSION_ID=w0t0p0:B2A638A3-C0C0-4FA1-BA7A-95DFF0A99BED
SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.rii0JhficH/Listeners
LC_TERMINAL_VERSION=3.5.9
COLORFGBG=11;15
ITERM_PROFILE=Default
SQLITE_EXEMPT_PATH_FROM_VNODE_GUARDS=/Users/vanessa/Library/WebKit/Databases
XPC_FLAGS=0x0
LANG=en_US.UTF-8
PWD=/Users/vanessa/dev/haskell/jacinda
SHELL=/bin/zsh
__CFBundleIdentifier=com.googlecode.iterm2
TERM_FEATURES=T3LrMSc7UUw9Ts3BFGsSyHNoSxF
TERM_PROGRAM_VERSION=3.5.9
TERM_PROGRAM=iTerm.app
PATH=/Users/vanessa/.opam/default/bin:/Users/vanessa/.cabal/bin:/Users/vanessa/.ghcup/bin:/Library/Frameworks/Python.framework/Versions/3.13/bin:/Library/Frameworks/Python.framework/Versions/3.12/bin:/Library/Frameworks/Python.framework/Versions/3.11/bin:/Library/Frameworks/Python.framework/Versions/3.10/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin:/opt/X11/bin:/Library/TeX/texbin:/usr/local/go/bin:/Users/vanessa/.cpm/bin:/Users/vanessa/pakcs/bin:/Users/vanessa/.local/bin:/Users/vanessa/.cargo/bin:/Applications/iTerm.app/Contents/Resources/utilities:/Users/vanessa/.fzf/bin
DISPLAY=/private/tmp/com.apple.launchd.YmA3AqJ9zk/org.xquartz:0
LC_TERMINAL=iTerm2
COLORTERM=truecolor
COMMAND_MODE=unix2003
TERM=xterm-256color
TERMINFO_DIRS=/Applications/iTerm.app/Contents/Resources/terminfo:/usr/share/terminfo
HOME=/Users/vanessa
TMPDIR=/var/folders/t5/6d5ws1yj0g78plhhzy_03ybm0000gn/T/
USER=vanessa
XPC_SERVICE_NAME=0
LOGNAME=vanessa
__CF_USER_TEXT_ENCODING=0x0:0:0
ITERM_SESSION_ID=w0t0p0:B2A638A3-C0C0-4FA1-BA7A-95DFF0A99BED
SHLVL=1
OLDPWD=/Users/vanessa/dev/haskell/apple
HISTSIZE=1000000000
HISTFILESIZE=1000000000
SAVEHIST=1000000000
OPAMNOENVNOTICE=true
OPAM_SWITCH_PREFIX=/Users/vanessa/.opam/default
CAML_LD_LIBRARY_PATH=/Users/vanessa/.opam/default/lib/stublibs:/Users/vanessa/.opam/default/lib/ocaml/stublibs:/Users/vanessa/.opam/default/lib/ocaml
OCAML_TOPLEVEL_PATH=/Users/vanessa/.opam/default/lib/toplevel
OPAM_LAST_ENV=/Users/vanessa/.opam/.last-env/env-e11ea89f0858dbd243348d81ca05c0a8-0
_=/usr/bin/printenv
