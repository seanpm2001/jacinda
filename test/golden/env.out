/Users/vanessa/.opam/default/bin:/Users/vanessa/.cabal/bin:/Users/vanessa/.ghcup/bin:/Library/Frameworks/Python.framework/Versions/3.13/bin:/Library/Frameworks/Python.framework/Versions/3.12/bin:/Library/Frameworks/Python.framework/Versions/3.11/bin:/Library/Frameworks/Python.framework/Versions/3.10/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin:/opt/X11/bin:/Library/TeX/texbin:/usr/local/go/bin:/Users/vanessa/.cpm/bin:/Users/vanessa/pakcs/bin:/Users/vanessa/.local/bin:/Users/vanessa/.cargo/bin:/Applications/iTerm.app/Contents/Resources/utilities:/Users/vanessa/.fzf/bin
